_targets.R

```diff
@@ -17,6 +17,7 @@ source("1_fetch/src/get_nwis_data.R")
 source("1_fetch/src/calc_HIT.R")
 source("1_fetch/src/calc_FDC.R")
 source("1_fetch/src/moving_window_functions.R")
+source("1_fetch/src/cross_validation_functions.R")
```

Comment:
Let's number this as `4_setup_crossval` and use p4 for the targets names. I think we'll add cross validation functions to this later to select CV groups

Reply:
Want this in a new 4_crossval folder, correct? Not in the fetch directory?

Reply:
yes, thanks!